### DISCUSSION TITLE
E-commerce tracking?

### DISCUSSION THREAD

**Discussion by CodeAndTonicHQ** [plausible/analytics#1687] [Anything Else]
Hi,

Trying out plausible after the GA GDPR news. Seems great. Quick painless and smooth. 

Is there an e-commerce tracking option? I of course can create a goal called 'Purchase' and add JS to order confiramtion page like:

`plausible('Purchase', {props: {value: '49'}});`

...but is there a way to create any kind of useful dashboard type of view, to see which utm campaigns brought which sales, filter by products sold etc? Is this available, planned, or possible with some kind of...third party thing? Can we connect third party services to plausible to drill down to the data?

**Update:** We've now opened up a preview version of this to everyone. You can now assign monetary values to custom events and track the revenue attribution of your ecommerce store. Instructions at https://plausible.io/docs/ecommerce-revenue-tracking

**Comment by CodeAndTonicHQ** [comment]
Ok, it seems that there's at least a paid connector between plausible and google data studio https://www.epilocal.com/products/plausible-data-studio-connector/

So maybe I could send the data to Google Data Studio and make e-commerce reporting there. I'd love more info, thoughts on future roadmap and how others have solved this? And if e-commerce tracking is or will be a focus. I think the goal tracking should be fine it's actullay lovely to implement. But how to use the goal data effectively for e-commerce

**Comment by metmarkosaric** [comment]
Thanks and glad you like Plausible, @CodeAndTonicHQ!

We'd love to improve the ecommerce side of things and introduce some pre-built reports for ecommerce. Take a look at this thread for details and do leave any feedback on what you'd like to see: https://github.com/plausible/analytics/discussions/1678

Our custom events and dynamic custom properties are powerful and flexible enough so some of the data you want can already be found using them. For instance, if you click on any specific custom event or property in your dashboard, we will filter the dashboard only by visitors who converted on that specific event/property. That will give you easy access to referral sources including UTM tags, entry pages, conversion pages, browsers etc with each of these metrics also giving you a conversion rate.

An example is here on our own live demo which I've filtered by visitors who signed up for a trial account: https://plausible.io/plausible.io?goal=Signup

And here's an example from the public dashboard of one of our customers who are using custom properties to track the price of the purchased item. I've filtered their dashboard by visitors who have paid $100 for their purchase https://plausible.io/elementary.io?goal=Payment&props=%7B%22Amount%22%3A%22100%22%7D

**Comment by metmarkosaric** [comment]
**Update:** We've now opened up a preview version of this to everyone. You can now assign monetary values to custom events and track the revenue attribution of your ecommerce store. Instructions at https://plausible.io/docs/ecommerce-revenue-tracking
